frontend/src/features/properties/map/research/ResearchContainer.tsx

```diff
@@ -167,12 +170,12 @@ export const ResearchContainer: React.FunctionComponent<IResearchContainerProps>
         <ResearchFileLayout
           leftComponent={
             <>
-              {selectedMenuIndex === 0 && (
+              {selectedMenuIndex === 0 && hasClaim(Claims.RESEARCH_ADD) ? (
```

Comment:
quick question - as per PSP-3833

> EXPECTED RESULTS: The user with functional (restricted) role should only be able to create research files with already mapped properties or create research files without linked properties.

With this if you have the research add then you can add/edit properties right? Do we have a specific claim for adding properties to a research file upon creation / edit?

Reply:
No, we don't.

Reply:
The idea with the claims is to provide something that is functionally equivalent. In this case, the ability to add a research file allows a user to create new properties of interest by attaching those properties to a research file. Similiarly, a user would be creating new POIs when adding new properties to an existing research file. Either way (adding new or updating existing) new POIs are created.